medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/DocumentController.cs

```diff
@@ -42,27 +43,40 @@ public DocumentController(DocumentManager.DocumentManagerClient documentManagerC
         }
 
         [HttpGet("MyDmers")]
-        [ProducesResponseType(typeof(IEnumerable<CaseDocument>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<DmerDocument>), 200)]
         [ProducesResponseType(401)]
         [ProducesResponseType(500)]
         public async Task<IActionResult> GetMyDocumentsByType()
         {
-            var profile = await _userService.GetCurrentUserContext();
-            var loginIds = profile.LoginIds;
+            var result = new List<DmerDocument>();
 
-            var dmerDocumentTypeCode = _configuration["Constants:DmerDocumentTypeCode"];
-            var request = new GetDocumentsByTypeForUsersRequest { DocumentTypeCode = dmerDocumentTypeCode, LoginIds = { loginIds } };
-            var reply = _documentManagerClient.GetDocumentsByTypeForUsers(request);
-            if (reply.ResultStatus == Rsbc.Dmf.CaseManagement.Service.ResultStatus.Success)
+            try
             {
-                var caseDocuments = _mapper.Map<IEnumerable<CaseDocument>>(reply.Items);
-                return Ok(caseDocuments);
+                var profile = await _userService.GetCurrentUserContext();
+                var loginIds = profile.LoginIds;
+
+                var dmerDocumentTypeCode = _configuration["Constants:DmerDocumentTypeCode"];
+                var request = new GetDocumentsByTypeForUsersRequest { DocumentTypeCode = dmerDocumentTypeCode, LoginIds = { loginIds } };
+                var reply = _documentManagerClient.GetDocumentsByTypeForUsers(request);
+                if (reply.ResultStatus == Rsbc.Dmf.CaseManagement.Service.ResultStatus.Success)
+                {
+                    var caseDocuments = _mapper.Map<IEnumerable<DmerDocument>>(reply.Items);
+                    return Ok(caseDocuments);
+                }
+                else
+                {
+                    _logger.LogError($"{nameof(GetMyDocumentsByType)} error: unable to get documents by type - {reply.ErrorDetail}");
+                    return StatusCode(500, reply.ErrorDetail);
+                }
             }
-            else
+            catch(Exception ex)
             {
-                _logger.LogError($"{nameof(GetMyDocumentsByType)} error: unable to get documents by type - {reply.ErrorDetail}");
-                return StatusCode(500, reply.ErrorDetail);
+                _logger.LogError(ex, $"Error getting DMER's");
+                return StatusCode(500, "Bad Request");
```

Comment:
not gonna nitpick here either but this is not a "Bad Request", but meh, who cares